torchvision/prototype/transforms/_geometry.py

```diff
@@ -252,7 +252,14 @@ def __init__(
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         fill = self.fill[type(inpt)]
-        return F.pad(inpt, padding=self.padding, fill=fill, padding_mode=self.padding_mode)
+
+        # This cast does Sequence[int] -> List[int] and is required to make mypy happy
```

Comment:
Well, `mypy` is not at fault here. The kernel could support `Sequence[int]`, but we annotated `List[int]`. Since that is more specific, `mypy` is right to complain. We can always silence `mypy` with a `# type: ignore[...]` comment or use `cast(...)` since both have no runtime implications. Actually casting to a list while it is not needed hurts performance although probably not by a significant amount.

Reply:
At this point where we try to improve the speed by any margin, I would prefer using ignore statements rather than casting. I don't know how negligible the introduction of `cast()` is but if there is no measurable slow-down I don't mind switching.